backend/vouch/tests/test_api_voucher.py

```diff
@@ -22,27 +24,24 @@ def setUp(self):
             "value": "1",
         }
 
-    def test_anonymous_cant_create(self):
+    def test_anon_401_create(self):
```

Comment:
@GresilleSiffle I fail to see how these method names are better. They look much more cryptic to me.

Reply:
Yes it's very subjective indeed.

I sometimes do some experimentations with names to see if I'm able to navigate easily in the tests.

The original name `test_anonymous_cant_create` is nearly perfect, and very human readable. I started to use `anonymous` and `authenticated` few months ago to make the target of the test explicit.

But quickly these names became a bit long. I started then to replace `anonymous` by `anon`, `authenticated` by `auth` (sometimes `users` when the test is interacting with both). It saved few characters.

Then some other parts of the name caused me troubles. Sometimes we write `cant`, sometimes `must_not`, sometimes `should_not`, and it makes tests with the same intention look different.

To continue experimenting I finally decided to use the HTTP code directly in the test name, to avoid having to interpret "should" and "cant".

 Now I find these names more concise, and they allow me to quickly see in a test case which tests are missing. It's true that `test_anon_401_create` is not that explicit at the first glance, but unless it causes more problems that it solves I'd like to propose us to continue with it, and to see in few weeks how we feel with these names.